bin/netlify-wait-for-build

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+current_commit=$(git rev-parse HEAD)
+
+echo "Running on commit $COMMIT_SHA"
+
+function get_status() {
+  echo "::group::API Response"
+  curl --header "Authorization: Bearer $GITHUB_TOKEN" "https://api.github.com/repos/actualbudget/actual/commits/$COMMIT_SHA/statuses" > /tmp/status.json
```

Comment:
I think this won't work on PRs coming from forks (so basically every other contributor). They won't have access to our `GITHUB_TOKEN`

Reply:
Maybe I'm totally wrong tough. Mind testing it out from a fork?

Reply:
I can change it to `pull_request_target` which uses the workflow file from the `master` branch and therefore always has access to the token.